src/components/dumb/OeContainer.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="container">
+    <div data-cy="container-content" class="container__content">
+      <slot></slot>
+    </div>
+    <div v-if="props.tabs" class="bottom-tabs" data-cy="bottom-tabs">
+      <vl-action-group>
+        <vl-pill
+          v-for="item in props.tabs"
+          :key="item.id"
+          :class="[{ 'is-active': item.id === props.activeTab?.id }, `tab-${item.id}`]"
+          :mod-closable="item.closable"
+          :close-label="`Sluit tab ${item.label}`"
```

Comment:
deze doet niks denk ik

Reply:
Deze rendered een extra element in de DOM voor screenreaders:
Zie: https://642e92e0cda6c627a0601f07-bpunzfpjnm.chromatic.com/?path=/docs/atoms-vl-pill--docs#accessibility